bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2217,7 +2201,7 @@ namespace bftEngine
 		{
 			Assert(newStateCheckpoint % checkpointWindowSize == 0);
 
-			LOG_INFO_F(GL, "onTransferringCompleteImp with newStateCheckpoint==%" PRId64 "", newStateCheckpoint);
+			LOG_DEBUG_F(GL, "onTransferringCompleteImp with newStateCheckpoint==%" PRId64 "", newStateCheckpoint);
```

Comment:
If we leave the message about starting state transfer at INFO, we'll likely want to leave this at INFO as well.

Reply:
I will do so.